testsuite/databases/rabbitmq/pytest_plugin.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from . import classes
+from . import service
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup('rabbitmq')
+    group.addoption('--rabbitmq')
+    group.addoption(
+        '--no-rabbitmq',
+        help='Disable use of RabbitMQ',
+        action='store_true',
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers',
+        'rabbitmq: per-test RabbitMQ initialization',
+    )
+
+
+def pytest_service_register(register_service):
+    register_service('rabbitmq', service.create_rabbitmq_service)
+
+
+@pytest.fixture
+def rabbitmq(
```

Comment:
i intentionally left this as-of-now-noop-fixture here because it might be convenient 
to implement per-test/static entities setup at some point, and then this fixture would actually do something useful